bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/InputDataFormattingTest.groovy

```diff
@@ -13,15 +13,14 @@ class InputDataFormattingTest {
 
     def h2GISDatabase
 
-    public static communeToTest = "12174"
+    public static communeToTest = "abcde"
```

Comment:
?

Reply:
I just replaced my insee code by this one, to be more "generic" and to remove as many initial informations as possible